src/bthome_ble/parser.py

```diff
@@ -710,13 +718,18 @@ def _decrypt_bthome(
             )
             raise ValueError
         if decrypted_payload is None:
-            self.bindkey_verified = False
+            if self.decryption_failed is True:
+                # we only ask for reautentification after the decryption has failed twice.
+                self.bindkey_verified = False
+            else:
+                self.decryption_failed = True
```

Comment:
Looking at the linked issue that had a problem with decryption and not an empty decrypted payload, the verbosity of the message here (error instead of warning, which makes sense because we don't expect an empty payload after decryption) and that there is no test coverage for this part, maybe we can start adding the logic only if `InvalidTag` is raised? If we encounter something similar with an empty payload we can add it later.

Reply:
Thanks for the review. 

Your comment makes sense, I've reverted the change in case of an empty payload.